gnocchi/carbonara.py

```diff
@@ -348,6 +348,13 @@ def __eq__(self, other):
                 and self.block_size == other.block_size
                 and self.back_window == other.back_window)
 
+    def _set_values(self, values):
```

Comment:
I see. I did not want to call a method with "_", which indicates an internal method. That is why I prepared the set_values method to handle situations where we do not handle any value in the set, and just trucate the series.

Do you prefer the otherway around? I mean, calling directly the "_truncate"?

Reply:
i think it's cleaner to explicitly call truncate rather than have a edge case where if you want to truncate you pass in None to set_values.

i'm indifferent as to whether we rename _truncate to truncate. i'm pretty lazy with honouring "non-public" methods in python, especially if they're within same package. i'm pretty sure there's code that does this already :)

Reply:
Ok, I will do that.